cocinaborcelle/src/clientApi/cardapio/getCardapio.ts

```diff
@@ -0,0 +1,16 @@
+import { Cardapio, Categoria, Comida } from "@prisma/client"
+import  api  from "../api"
+
+type CardapioFull = {
+    categoria: Categoria,
+} & Comida[]
```

Comment:
```suggestion
type CardapioFull = Cardapio & {
    categorias: (Categoria & { comidas: Comida[] })[];
};
```

recomendo vc rodar "npm run dev" e colocar na barra de navegação do browser ou no thunder client a url "http://localhost:3000/api/cardapio/index". Você vai ver o que essa api tá retornando e poder tipar este retorno. Ao examinar, vc poderá ver que o tipo é esse